sdk/metrics/src/jmh/java/io/opentelemetry/sdk/metrics/internal/aggregator/HistogramAggregationParam.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.aggregator;
+
+import io.opentelemetry.sdk.metrics.exemplar.ExemplarReservoir;
+import java.util.Collections;
+
+/** The types of histogram aggregation to benchmark. */
+@SuppressWarnings("ImmutableEnumChecker")
+public enum HistogramAggregationParam {
+  EXPLICIT_DEFAULT_BUCKET(
+      new DoubleHistogramAggregator(
+          ExplicitBucketHistogramUtils.createBoundaryArray(
+              ExplicitBucketHistogramUtils.DEFAULT_HISTOGRAM_BUCKET_BOUNDARIES),
+          ExemplarReservoir::noSamples)),
+  EXPLICIT_SINGLE_BUCKET(
+      new DoubleHistogramAggregator(
+          ExplicitBucketHistogramUtils.createBoundaryArray(Collections.emptyList()),
+          ExemplarReservoir::noSamples)),
+  EXPONENTIAL(new DoubleExponentialHistogramAggregator(ExemplarReservoir::noSamples));
```

Comment:
Eventually I expect there will be some configuration parameter for max buckets / expected range (like in the recent go implementation), which will to some extent affect the benchmark. Once that exists, it will be interesting to see how those parameters affect performance.

Reply:
Yep!  I'd love to have those as well.